docs/reference/schema.md

```diff
@@ -10,14 +10,14 @@ This page presents the schema in tables with additional information in paragraph
    If any conflicts are found between the text on this page and the text within the schema, the text within the schema takes precedence.
 ```
 
-The RDLS schema covers [dataset attributes](#dataset), [resource attributes](#resource) and four risk-specific components:
+The RDLS schema covers [dataset attributes](#dataset), [resource attributes](#resource) and four components to document data used in risk analysis:
 
-- [Hazard](#hazard): main hazard type, specific process, trigger of the hazard, occurrence frequency of event, intensity unit to measure the process and analytical method.
-- [Exposure](#exposure): asset category, specific taxonomy, cost type and value.
-- [Vulnerability](#vulnerability): model that links hazard intensity and exposure classification to measure of impact over the total exposed value.
-- [Loss](#loss): modelled damage and losses produced in a risk assessment as a function of hazard, exposure and vulnerability components.
+- [Hazard](#hazard): including the main hazard type and process, triggering hazard and process, hazard intensity units, occurrence frequency of individual events, multiple hazard footprints per event, historical and stochastic events sets, and analytical methods used.
+- [Exposure](#exposure): including asset category (including human, built environment assets and natural assets), taxonomy used to describe characteristics of assets and people, cost type, location, geometry types.
+- [Vulnerability](#vulnerability): describing vulnerability and fragility relationships and indexes, which estimate the impact on people or assets as a function of hazard intensity. This component uses attributes consistent with the hazard, exposure and loss components.
+- [Loss](#loss): modelled monetary and non-monetary damage and losses produced in a risk assessment, including the ability to link losses to the hazard, exposure, and vulnerability components used in the analysis. Allows description of common impact and risk metrics for direct and indirect impacts, and for individual historical and hypothetical events, and for large events sets in year loss tables and event loss tables.
```

Comment:
You've described these 4 components in 2 different ways. Hazard and Exposure are essentially just listing the fields and objects within the components. Vulnerability and Loss are more descriptive as to what these components represent and what the actual data is which could be read as implying that the schema covers the data. It doesn't, it covers the metadata that describes the data.

I'd suggest picking one type of description and using that for all 4. Given that the Glossary contains (or will contain) definitions for all 4 component concepts I'd recommend using this section as a description of what the components contain, i.e. the details of what data can be described using each component.

Reply:
Resolved in ad586d2 commit. Also made other descriptions shorter and more targeted.